src/db/spanner/models.rs

```diff
@@ -777,18 +766,17 @@ impl SpannerDb {
     }
 
     pub fn delete_storage_sync(&self, user_id: params::DeleteStorage) -> Result<()> {
-        let params = params! {
-            "fxa_uid" => user_id.fxa_uid.clone(),
-            "fxa_kid" => user_id.fxa_kid.clone()
-        };
         // Also deletes child bso rows (INTERLEAVE IN PARENT user_collections
         // ON DELETE CASCADE
         self.sql(
             "DELETE FROM user_collections
               WHERE userid = @fxa_uid
                 AND fxa_kid = @fxa_kid",
         )?
-        .params(params.clone())
+        .params(params! {
+            "fxa_uid" => user_id.fxa_uid,
+            "fxa_kid" => user_id.fxa_kid,
+        })
```

Comment:
This was more me trying to be consistent about how params were being defined for functions. The clone was definitely unneeded. Thanks!

Reply:
No worries, I think this is actually leftover from removing a second unneeded DELETE query anyway